diff --git a/src/gpu/ganesh/gl/GrGLAssembleWebGLInterfaceAutogen.cpp b/src/gpu/ganesh/gl/GrGLAssembleWebGLInterfaceAutogen.cpp
index a46b65f..6b7f560 100644
--- a/src/gpu/ganesh/gl/GrGLAssembleWebGLInterfaceAutogen.cpp
+++ b/src/gpu/ganesh/gl/GrGLAssembleWebGLInterfaceAutogen.cpp
@@ -11,7 +11,7 @@
 #include "include/core/SkRefCnt.h"
 #include "include/gpu/ganesh/gl/GrGLAssembleInterface.h"
 
-#if SK_DISABLE_WEBGL_INTERFACE || !defined(__EMSCRIPTEN__)
+#if SK_DISABLE_WEBGL_INTERFACE
 struct GrGLInterface;
 sk_sp<const GrGLInterface> GrGLMakeAssembledWebGLInterface(void *ctx, GrGLGetProc get) {
     return nullptr;
